src/utils/ocr/read.py

```diff
@@ -50,7 +50,19 @@ def image_to_text(img: np.ndarray, line_boxes: bool = False, do_pre_proc: bool =
     if img is None or len(img) == 0:
         LOGGER.warning("img provided to image_to_text() is empty!")
         return OcrResult("", "", word_confidences=0, mean_confidence=0), [] if line_boxes else ""
-    final_img = img
+
+    # Apply a border to the image. This weird hack prevents the "Error in boxClipToRectangle" errors.
+    # Read more here: https://github.com/tesseract-ocr/tesseract/issues/427#issuecomment-248153491
+    border_size = 10
```

Comment:
did you look at the performance hit when using this code?

Reply:
I had not, and I should have because I knew you'd ask. I just ran it from source 6 times, the first 3 are run from main, the next 3 are from this branch.

Average of main find_descr: 0.2453215016092276
Average of branch find_descr: 0.2194741764064181

Average of main read_descr: 0.3181950116090013
Average of branch read_descr: 0.2986032384367095

So based on this, it's actually slightly faster? I wonder if that's just a fluke or if the border lets it read text faster. Either way doesn't seem to be a significant increase if any

Runs from main:
2024-06-29 | 17:47:54.637 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.2714955149916932, 'max': 0.3457575000356883, 'median': 0.2633587000891566, 'min': 0.23023540002759546}
2024-06-29 | 17:47:54.637 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.32491972307280564, 'max': 0.3923218999989331, 'median': 0.3217535999137908, 'min': 0.293472700053826}

2024-06-29 | 17:52:28.353 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.2462864136587913, 'max': 0.32281790010165423, 'median': 0.2446953000035137, 'min': 0.21637560008093715}
2024-06-29 | 17:52:28.353 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.30513109999446897, 'max': 0.31848959997296333, 'median': 0.30556459992658347, 'min': 0.29461890005040914}

2024-06-29 | 17:57:18.876 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.21818257617719827, 'max': 0.2453525000018999, 'median': 0.21564429998397827, 'min': 0.20646469993516803}
2024-06-29 | 17:57:18.876 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.3245342117597294, 'max': 0.34792720002587885, 'median': 0.32352900004480034, 'min': 0.3002644000807777}

Runs from branch:
2024-06-29 | 18:05:38.061 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.21831184399314224, 'max': 0.27271719998680055, 'median': 0.21525400003883988, 'min': 0.2011806999798864}
2024-06-29 | 18:05:38.061 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.2960329444354607, 'max': 0.3149370000464842, 'median': 0.29711659997701645, 'min': 0.28346579999197274}

2024-06-29 | 18:08:36.267 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.21574446521496968, 'max': 0.2387196000199765, 'median': 0.21321349998470396, 'min': 0.2016857999842614}
2024-06-29 | 18:08:36.267 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.3087488764499807, 'max': 0.3494996998924762, 'median': 0.30607619998045266, 'min': 0.28394210000988096}

2024-06-29 | 18:13:44.216 | MainThread | INFO | __main__:157 | 2160p_1_find_descr: {'avg': 0.224366220011143, 'max': 0.29219649999868125, 'median': 0.2173699999693781, 'min': 0.2049325000261888}
2024-06-29 | 18:13:44.217 | MainThread | INFO | __main__:157 | 2160p_1_read_descr: {'avg': 0.29102789442468846, 'max': 0.3381747000385076, 'median': 0.28838400007225573, 'min': 0.2790110999485478}